src/main/java/org/deepsymmetry/beatlink/Util.java

```diff
@@ -360,6 +361,20 @@ public static long bytesToNumber(byte[] buffer, int start, int length) {
         return result;
     }
 
+    /**
```

Comment:
What is this for?

Reply:
I use this to compare signed and unsigned byte arrays. Performance is very quick and it is much easier to debug/compare Strings (rather than byte arrays which I find to be more difficult to work with in the IDE).

Reply:
Hmm, that does not sit quite right with me. Surely this is a use case for `BigInteger`, or perhaps `ByteBuffer`, which implements `Comparable`?

Reply:
Done, seems to work great. Thanks!